public/docs/ts/latest/cookbook/i18n.jade

```diff
@@ -87,17 +90,113 @@ a#i18n-attribute
+:marked
+  ## Add _i18n-..._ translation attributes
+  Suppose you add an image to your html. You care about accessibility so you add a `title` tag as well:
+
++makeExample('cb-i18n/ts/app/app.component.1.html', 'i18n-title', 'app/app.component.html')(format=".")
+
+:marked
+  The `title` attribute needs to be translated. 
+  Angular i18n support has more translation attributes in the form,`i18n-x`, where `x` is the
+  name of the attribute to translate. 
+  
+  To translate the `title` on the `img` tag from the previous example, write:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-title-translate', 'app/app.component.html')(format=".")
+
+:marked
+  This attribute also supports the notation `i18n-x="<meaning>|<description>"`.
+
+.l-main-section
+a#cardinality
+:marked
+  ## Handle singular and plural
+  
+  Different languages have different pluralization rules.
+  
+  Suppose your application says something about a collection of wolves. 
+  In English, depending upon the number of wolves, you could display "no wolves", "one wolf", "two wolves", or "a wolf pack".
+  Other languages might express the _cardinality_ differently. 
+  
+  Here's how you could markup the component template to display the phrase appropriate to the number of wolves:
+
++makeExample('cb-i18n/ts/app/app.component.html', 'i18n-plural', 'app/app.component.html')(format=".")
+
+:marked
+  * The first parameter is the key. It will be bound to the component property (`wolves`) that determines the number of wolves.
+  * The second parameter indentifies this as a `plural` translation type.
+  * The third parameter defines a pluralization pattern consisting of pluralization categories and their matching values.
+
+  Pluralization categories include:
+  * =0 / zero
+  * =1 / one
+  * =5 / five
+  * few
+  * other
+
+  Put the default _English_ translation in braces (`{}`) next to the pluralization category.
+  * When you're talking about one wolf, you could write `=1 {one wolf}`.
+  * For zero wolves, you could write `=0 {no wolves}`.
+  * For two wolves, you could write `=2 {two wolves}`. 
+
+  You could keep this up for three, four, and every other number of wolves.
+  Or you could specify the **`other`** category as a catch-all for any unmatched cardinality
+  and write something like: `other {a wolf pack}.  
+
+.l-sub-section
+  :marked
+    This syntax conforms to the
+    <a href="http://userguide.icu-project.org/formatparse/messages" target="_blank" title="ICU Message Format">ICU Message Format</a>
+    which derives from the 
+    <a href="http://cldr.unicode.org/" target="_blank" title="CLDR">Common Locale Data Repository (CLDR)</a>
+    which specifies the
+    <a href="http://unicode.org/reports/tr35/tr35-numbers.html#Language_Plural_Rules" target="_blank" title="Pluralization Rules">pluralization rules</a>.
+
+    The category `=1` matches the count of `1` exactly. 
+    The category `one` matches any count with a `1` digit in it such as `1` or `21`, 
```

Comment:
Do we really need this in the docs? The pluralization rules just get weirder and weirder the more you  read about them. Our docs don't seem like the right place for this.